draft-irtf-cfrg-opaque.md

```diff
@@ -720,19 +718,15 @@ data
 ~~~
 struct {
     opaque data<1..2^16-1>;
-    opaque pkS<1..2^16-1>;
-    Envelope envelope;
+    Envelope envU;
 } CredentialResponse;
 ~~~
 
 data
 : A serialized OPRF group element.
 
-pkS
-: An encoded public key that will be used for the online authenticated key exchange stage.
-
```

Comment:
pkS should not be removed -- see my last comment on #63 . The problem is that the pkS is no longer included in the envelope.